content/blog/2017/07/2017-07-07-jenkins-conan.adoc

```diff
@@ -0,0 +1,221 @@
+---
+layout: post
+title: "Continuous Integration for C/C++ Projects with Jenkins and Conan"
+tags:
+- event
+- Jenkins User Conference
+- Jenkins Community Day Paris
+author: alyssat
+---
+
+NOTE: This is a guest post by Luis Martínez de Bartolomé, Conan Co-Founder
```

Comment:
"Conan" should include a link to something which will explain what Conan actually is